bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/LimitsListener.java

```diff
@@ -83,7 +122,7 @@ public void propertyChange(PropertyChangeEvent evt) {
 
 			}
 
-            // LCH Laser Shutter Warnings
+			// LCH Laser Shutter Warnings
```

Comment:
Something funky happened with the comment indentation here.

Reply:
QPT is formatted with tabs so we should probably do a PR to reformat everything.

Reply:
It's my fault. I used to think it was a good idea.

Reply:
There was already a mix of spaces of tabs on that file, so I used spaces for new editions.

I didn't want to change everything to spaces in this PR because it would be difficult to review, but we should create a separate PR just for replacing all the tabs.